src/Compilers/CSharp/Portable/Symbols/Attributes/AttributeData.cs

```diff
@@ -232,7 +232,7 @@ internal bool IsSecurityAttribute(CSharpCompilation compilation)
 
         #region Attribute Decoding
 
-        internal void DecodeSecurityAttribute<T>(Symbol targetSymbol, CSharpCompilation compilation, ref DecodeWellKnownAttributeArguments<AttributeSyntax, CSharpAttributeData, AttributeLocation> arguments)
+        internal void DecodeSecurityAttribute<T>(Symbol targetSymbol, CSharpCompilation compilation, ref DecodeWellKnownAttributeArguments<AssemblySymbol, AttributeSyntax, CSharpAttributeData, AttributeLocation> arguments)
```

Comment:
note: a using alias for this type would make a lot of this simpler (see later PRs for an example of this).  However, i wanted to do the minimal amount of effort here to keep the review minimal.